### ISSUE TITLE
Login Facebook wrong

### ISSUE THREAD

**Issue by overpress** [Automattic/facebook-instant-articles-wp#145]
Hi, 

we don't understand if the login was successful. We choose the page but then calls again to log in. What should we do?

See this video -> https://dl.dropboxusercontent.com/u/22334399/Problem%20FB%20Instant%20Article.mov

Thanks

**Comment by diegoquinteiro** [comment]
Hi, @overpress. Thanks for the feedback.

Could you provide some more information about the issue you're having? That will help us reproduce the issue.

- What version of the plugin are you using?
- What version of WordPress are you using?
- What version of PHP are you using (if known)?
- What's the debug output shown in the post edit screen on the Facebook Instant Article box?

Also, can you change WP_DEBUG to true on your wp_config.php file and share any warning/error shown after this?

Thank you again!

**Comment by overpress** [comment]
Yes,

Plugin version -> 2.7
WordPress version -> 4.5
PHP version -> We don't know

What's the debug output shown in the post edit screen on the Facebook Instant Article box? -> Where I find it?

Debug on show ->

```
Notice: Il metodo del costruttore richiamato per WP_Widget in ARFwidgetForm è deprecato dalla versione 4.3.0! Utilizzare invece
__construct()
. in /home/overpress/public_html/wp-includes/functions.php on line 3722

Strict Standards: Non-static method arformcontroller::checksorting() should not be called statically, assuming $this from incompatible context in /home/overpress/public_html/wp-content/plugins/arforms/core/controllers/maincontroller.php on line 602

Strict Standards: Non-static method armainhelper::frm_capabilities() should not be called statically, assuming $this from incompatible context in /home/overpress/public_html/wp-content/plugins/arforms/core/models/arsettingmodel.php on line 306

Notice: Undefined variable: POST_CATS in /home/overpress/public_html/wp-content/plugins/arscode-ninja-popups/arscode-ninja-popups.php on line 1668


```

**Comment by overpress** [comment]
We discovered another problem: If we activate your plugin we can no longer publish new articles and new pages.

When we open the pages http://overpress.it/wp-admin/post-new.php and http://overpress.it/wp-admin/post-new.php?post_type=page there returns error 404.

**Comment by diegoquinteiro** [comment]
@overpress related to this new problem, is there any other message shown when you turn WP_DEBUG to true?

**Comment by overpress** [comment]
The problems we had reported were all solved with the new version 2.8. Now everything works.

I wanted to ask another thing: How can we change the style of the post? We wanted to add the category to the article and add the like button to the right of the logo. How can we do?
